mappings/net/minecraft/world/chunk/ChunkToNibbleArrayMap.mapping

```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/class_3556 net/minecraft/world/chunk/ChunkToNibbleArrayMap
+	FIELD field_15789 cachePositions [J
+	FIELD field_15790 cacheArrays [Lnet/minecraft/class_2804;
+	FIELD field_15791 arrays Lit/unimi/dsi/fastutil/longs/Long2ObjectOpenHashMap;
+	FIELD field_16447 hasCache Z
```

Comment:
`cached`

Reply:
Changed it to `cacheEnabled` (it's true if the map is allowed to use a cache,and set to false by `disableCache`).